toolchain/gsl.py

```diff
@@ -46,9 +46,9 @@ def setUp(self):
     def test(self):
         process.run("make -k check", ignore_status=True, sudo=True)
         logfile = os.path.join(self.logdir, "stdout")
-        if process.system_output("grep -Eai 'FAIL:  [1-9]' %s" %
+        if process.system_output("grep -Eai 'FAIL: [1-9]' %s" %
```

Comment:
can we have regex pattern to grep this line as that will work for any kind of space pattern